sdk/tests/conformance2/transform_feedback/transform_feedback.html

```diff
@@ -114,6 +116,38 @@
     wtu.glErrorShouldBe(gl, gl.NO_ERROR, "there should be no errors");
 }
 
+function runTFBufferBindingTest() {
+    debug("");
+    debug("Testing binding and unbinding transform feedback objects and buffers");
+
+    buf = gl.createBuffer();
+    gl.bindBuffer(gl.TRANSFORM_FEEDBACK_BUFFER, buf);
+    gl.bufferData(gl.TRANSFORM_FEEDBACK_BUFFER, 16, gl.STATIC_DRAW);
+    gl.bindBuffer(gl.TRANSFORM_FEEDBACK_BUFFER, null);
+
+    tf = gl.createTransformFeedback();
+    gl.bindTransformFeedback(gl.TRANSFORM_FEEDBACK, tf);
+    gl.bindBufferBase(gl.TRANSFORM_FEEDBACK_BUFFER, 0, buf);
+    gl.bindTransformFeedback(gl.TRANSFORM_FEEDBACK, null);
+
+    // gl.TRANSFORM_FEEDBACK_BUFFER is part of Transform Feedback objects'
+    // state. See OpenGL ES 3.0.5 Section 6.24.
+    //
+    // Since the TRANSFORM_FEEDBACK was just unbound, there should be nothing
+    // bound.
+    shouldBeNull('gl.getIndexedParameter(gl.TRANSFORM_FEEDBACK_BUFFER_BINDING, 0)');
+
+    // Binding the buffer to the ARRAY_BUFFER binding point should succeed.
+    gl.bindBuffer(gl.ARRAY_BUFFER, buf);
+    wtu.glErrorShouldBe(gl, gl.NO_ERROR, "binding buffer to ARRAY_BUFFER");
```

Comment:
"Buffers should not be bound or in use for both transform feedback and other
purposes in the GL. Specifically, if a buffer object is simultaneously bound to a
transform feedback buffer binding point and elsewhere in the GL, any writes to
or reads from the buffer generate undefined values. Examples of such bindings
include ReadPixels to a pixel buffer object binding point and client access to a
buffer mapped with MapBufferRange."

So yeah, it's way beyond just draw, and ARRAY_BUFFER is definitely problematic.